connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -199,33 +234,37 @@ public Object get(String key) {
         }
     }
 
+    public static ConfigDef BASE_CONFIGS = new ConfigDef()
```

Comment:
Don't make this a static field, it leaves opportunities for it to get mutated accidentally.